float32 lat
float32 lon
float32 z
float32 yaw_rate
string frame_id
bool update_frame
---
bool success
string message
